docs/aca/04-aca-dapr-stateapi/index.md

```diff
@@ -424,12 +425,12 @@ Until this moment Dapr was not enabled on the Container Apps we have provisioned
 az containerapp dapr enable --name $BACKEND_API_NAME `
                             --resource-group $RESOURCE_GROUP `
                             --dapr-app-id  $BACKEND_API_NAME `
-                            --dapr-app-port $API_APP_PORT
+                            --dapr-app-port $TARGET_PORT
```

Comment:
It is better to keep dapr app port variables separated between apps

Reply:
What is the rationale for that? Is the sidecar not entirely local to the container app? I'm happy to make an adjustment but want to understand the issue first.

Reply:
it will be 80 for all container apps in this workshop as we are using http protocol. should be fine to use 1 single variable.